finalPrep/README.md

```diff
@@ -44,31 +44,43 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here`  
+
+   <img width="940" alt="week1top" src="https://user-images.githubusercontent.com/100865025/161394525-b134cce7-04a9-4a6f-9d9a-283c6afef865.png">  
+
+   <img width="938" alt="week1bttm" src="https://user-images.githubusercontent.com/100865025/161394582-775a5bd6-0284-48c3-aec4-394b101db579.png">
+- `Add Week 2 Screenshot Here`  
+
+   <img width="976" alt="week2top" src="https://user-images.githubusercontent.com/100865025/161394593-3c8a022d-2dc1-40ae-bb48-a518321291ba.png">  
+
+   <img width="968" alt="week2bttm" src="https://user-images.githubusercontent.com/100865025/161394628-5d6ec19b-b74e-4ee0-9173-fcbaa1c4062d.png">
+- `Add Week 3 Screenshot Here`  
+
+   <img width="978" alt="week3top" src="https://user-images.githubusercontent.com/100865025/161394635-a37982c2-040c-4d88-bd8e-1c1f93f30930.png">  
+
+   <img width="974" alt="week3bttm" src="https://user-images.githubusercontent.com/100865025/161394640-49070299-f298-470a-87f0-a89ef3e657c3.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Mentorship](https://gist.github.com/JCookDev/141d2137e28afd96f8718ea1cf8405a1)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here:[Lesson Prep](https://gist.github.com/JCookDev/5c206dc6f965097e057d97aa6318fd7c)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here:[Group Work Prep](https://gist.github.com/JCookDev/312a97b2ef6bad09b7b84aa56895b7d0.js)
```

Comment:
I saw your note about timezones, glad you are thinking about that. I've noticed that instructors are working to pair students that are within 1 timezone, max of 2 timezones.